resolwe_bio/processes/icount/import_bed_icount.yml

```diff
@@ -0,0 +1,46 @@
+- slug: upload-bed-icount
+  name: BED track file (iCount)
+  requirements:
+    expression-engine: jinja
+    executor:
+      docker:
+        image: "resolwe/bio-linux8-resolwe-preview:icount"
+  data_name: 'iCount-BED ({{ src.file|default("?") }})'
+  version: 1.0.0
+  type: data:bed:icount
+  category: upload
+  flow_collection: sample
```

Comment:
Yes. Each sample is a demultiplexed FASTQ file. Then it is mapped with STAR. And then, crosslinks are quantified by xlsites process, producing `data:bed:icount` object, which represents a single sample. This process (import_bed_icount) is just an alternative way to import already processed sample into the platform. It also comes handy when testing the pipeline.